src/main/java/gui/App.java

```diff
@@ -77,33 +74,11 @@ public static FXMLLoader loadScene(String path) {
         return null;
     }
 
-    private static void setUpProperties() throws IOException {
-        properties = new CustomProperties();
-        try {
-            FileReader fileReader = new FileReader("properties.txt");
-            properties.load(fileReader);
-            fileReader.close();
-        } catch (FileNotFoundException e) {
-            System.out.println("Making the properties.txt file");
-            FileWriter fileWriter = new FileWriter("properties.txt");
-            properties.store(fileWriter, "Property files for Dynamite and Butterflies");
-            fileWriter.close();
-        } catch (IOException e1) {
-            e1.printStackTrace();
-        }
-    }
-
     @Override
     public void stop() {
-        properties.updateProperties();
-
-        String stringOfFile = properties.getProperty("file", "def");
-        int numOfBookmarks = Integer.parseInt(
-                        properties.getProperty("bookmarkNum" + stringOfFile, "-1"));
-        properties.setProperty("bookmarkNum" + stringOfFile, Integer.toString(numOfBookmarks));
-
-        properties.saveProperties();
-
+        String stringOfFile = prefs.get("file", "def");
+        int numOfBookmarks = prefs.getInt("bookmarkNum" + stringOfFile, -1);
+        prefs.putInt("bookmarkNum" + stringOfFile, numOfBookmarks);
```

Comment:
Please go back to actually using properties... Thomas Abdeel will not be able to run this. I can help if you want.